xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1171 @@
+    <para>
+     We provide <link xlink:href="https://github.com/openSUSE/helm-mirror">helm-mirror</link>
+     to allow downloading all charts present in a chart repository in bulk and
+     moreover to extract all container image URLs from the charts.
+     <link xlink:href="https://github.com/containers/skopeo">skopeo</link> is
+     used to download all the images referred to in the Helm charts from their
+     respective registry.
+    </para>
+    <para>
+     Helm charts will be provided to the internal network by a webserver and
+     refer to the container images hosted on the internal registry mirror.
+    </para>
+    <para>
+     Once mirroring is configured, you will not have to modify Dockerfile(s) or
+     &kube; manifests to use the mirrors. The requests are passed through
+     the container engine which forwards them to the configured mirrors. For
+     example: All images with a prefix <literal>registry.suse.com/</literal>
+     will be automatically pulled from the configured (internal) mirror instead.
+    </para>
+    <para>
+     For further information on registry mirror configuration, refer to
+     <xref linkend="sec.admin.velum.registry" /> and
+     <xref linkend="sec.admin.velum.mirror" />.
+    </para>
+   </sect3>
+  </sect2>
+
+  <sect2 xml:id="sec.deploy.scenarios.airgap.requirements">
+   <title>Requirements</title>
+   <sect3 xml:id="sec.deploy.scenarios.airgap.requirements.machines">
+    <title>Mirror Servers</title>
+    <note>
+     <title>Shared Mirror Server</title>
+     <para>
+      If you have multiple &productname; clusters or a very large amount of
+      nodes accessing the mirrors, you should increase the sizing of CPU/RAM.
+     </para>
+     <para>
+      Storage sizing depends on your intended update frequency and data
+      retention model. If you want to keep snapshots or images of repository
+      states at various points, you must increase storage size accordingly.
+     </para>
+    </note>
+    <para>
+     You will need to provide and maintain at least two machines in addition to
+     your &productname; cluster. These mirror servers will reside on the
+     external part of your network and the internal (airgapped) network
+     respectively.
+    </para>
+    <variablelist>
+     <varlistentry>
+      <term>External</term>
+      <listitem>
+       <para>
+        This machine will host the <literal>&rmtool;</literal> for RPM packages
+        and the <literal>container image registry</literal> for container
+        images.
+       </para>
+       <itemizedlist>
+        <listitem>
+         <para>
+          <literal>1</literal> Host machines for the mirror servers.
+         </para>
+         <itemizedlist>
+          <listitem>
+           <para>
+            SLES 15
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            2 (v)CPU
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            8 GB RAM
```

Comment:
nit: isn't 8G a bit too much for a node which only runs a registry and RMT? :)

Reply:
I have no idea. Who can tell me?

Reply:
It's probably a bit too much, but honestly I wouldn't bother. If you really want to go the extra mile you look at what are the requirements on the official RMT documentation

Reply:
OK fine by me.